ci/supportedSystems.nix

```diff
@@ -0,0 +1,7 @@
+[
+  "aarch64-linux"
+  "aarch64-darwin"
+  #"i686-linux"  # !!!
```

Comment:
Why is this commented out?

Reply:
I kept it around because it was there already, but I just tracked it down via https://github.com/NixOS/ofborg/commit/db5293a4fb219d69f5734d87af573e1940396587 to https://github.com/NixOS/ofborg/commit/998adc8e2a7fddf9610d10a7657fb34c55492332, looks like it can be removed!

Reply:
When building graphics drivers, we need to build the corresponding 32 bit package as well. Since eval is not as "costly" anymore, can we restore this?

Reply:
I'd say let's keep this PR focused on just mirroring what ofborg/Hydra do, especially because we need to see how GitHub Actions handles it for Nixpkgs volume.

Reply:
Let's remove it then.